### PR TITLE
update getting started guide

### PR DESCRIPTION
**Pull request by senior7515**
@shinjikim  FYI.

We need to do cluster commands.

### CODE REVIEW

File: source/getting_started.rst [hunk 1]
```diff
@@ -0,0 +1,528 @@
...
+
+.. note::
+
+   Get up and running in 3 steps!
+
+   $ docker pull concord/client_devbox
+
+   $ docker run -i -t concord/client_devbox
+
+   $ ./cluster_start.bash
```
Comment (bigs) [inline source/getting_started.rst]: leave a note -- the cluster is now listening on X port with mesos, zk and concord running

File: source/getting_started.rst [hunk 2]
```diff
@@ -0,0 +1,528 @@
...
+
+
```
Comment (bigs) [inline source/getting_started.rst]: whitespaceeeeeeeee

File: source/getting_started.rst [hunk 3]
```diff
@@ -0,0 +1,528 @@
...
+
+
+Ruby Example
+============
+
+WordCounter
+-----------
+
```
Comment (bigs) [inline source/getting_started.rst]: WS!

Comment (bigs) [inline source/getting_started.rst]: also, there's no real lead in to this -- you just jump into an example. maybe a paragraph explaining the problem

File: source/getting_started.rst [hunk 4]
```diff
@@ -0,0 +1,528 @@
...
+*metadata* function.
+
+.. code:: ruby
+
+   Rbonut::Metadata.new(name: 'word-counter',
+                        istreams: [['words', Rbonut::StreamGrouping::GROUP_BY]],
+                        ostreams: ['word-counts'])
+
+
+One of the many ways in which Concord stands out from other stream processors
```
Comment (bigs) [inline source/getting_started.rst]: condense this para and the next two (ending with "to emit messages on them"). this was way too marketing-y -- my bad.

style should be more objective, straightforward. explain that existing systems require static topology, we don't. this makes ad-hoc deploys possible & saves you the trouble of tearing down working components. then lead into the metadata bit.

File: source/getting_started.rst [hunk 5]
```diff
@@ -0,0 +1,528 @@
...
+The default routing strategy is *ROUND_ROBIN*. By declaring an incoming stream
+as a string (rather than a string, routing strategy pair), it will default to
+*ROUND_ROBIN* grouping. To explicitly define a routing strategy, each stream should
+be represented by a pair *[name, grouping]* (note: in Ruby, there is no native
+pair type, so we represent it by a list with two item.)
+
+Returning to the code sample, note that we initialize a hash map to store our
+counts for each individual word:
+
+.. code:: ruby
```
Comment (bigs) [inline source/getting_started.rst]: `code-block` is necessary to get syntax highlighting

File: source/getting_started.rst [hunk 6]
```diff
@@ -0,0 +1,528 @@
...
+      end
+    end
+
+    Rbonut::Computation.serve(WordCounter.new)
+
+
+There are a few things going on, here. First, let's focus on the contents of the
+*metadata* function.
+
+.. code:: ruby
```
Comment (bigs) [inline source/getting_started.rst]: all `code` directives should be `code-block`

File: source/getting_started.rst [hunk 7]
```diff
@@ -0,0 +1,528 @@
...
+- *stream*: the globally unique name of the stream to emit a record on
+- *key*: the key to feed into the routing strategy. Not all operators will need
+  to emit a key, in which case it can be left as an empty string.
+- *value*: a binary blob of data (structured however you prefer) to be emitted
+  down stream.
+
+
+WordGenerator
+-------------
+
```
Comment (bigs) [inline source/getting_started.rst]: WS

File: source/getting_started.rst [hunk 8]
```diff
@@ -0,0 +1,528 @@
...
+random word on the stream 'words' every 100ms. Note that *WordGenerator* has no
+input streams, as it is a data source - only output streams.
+
+
+Putting it all together
+========================
+
+.. note::
+
+   if you get stuck, feel free to visit our repo for this demo
```
Comment (bigs) [inline source/getting_started.rst]: if -> If

File: source/getting_started.rst [hunk 9]
```diff
@@ -0,0 +1,528 @@
...
+
+   https://github.com/concord/rb-gettintg-started.git
```
Comment (bigs) [inline source/getting_started.rst]: typo in this link, also we should make this a proper link in the style used elsewhere

visit our repo_

.. _repo: link

File: source/getting_started.rst [hunk 10]
```diff
@@ -0,0 +1,528 @@
...
+
```
Comment (bigs) [inline source/getting_started.rst]: WS

File: source/getting_started.rst [hunk 11]
```diff
@@ -0,0 +1,528 @@
...
+
+
+
+First: Directory structure:
+----------------------------
+
+.. code:: bash
+
+        rb-gettintg-started $ tree
```
Comment (bigs) [inline source/getting_started.rst]: gettintg -> getting

File: source/getting_started.rst [hunk 12]
```diff
@@ -0,0 +1,528 @@
...
+          "executable_name": "runner.bash word_counter.rb",
+          "compress_files": ["word_counter.rb", "Gemfile",
+                             "Gemfile.lock", "runner.bash"],
+          "computation_name": "word-counter"
+        }
+
+
+.. note::
+
+   There are all the options you can use. please visite the CLI_
```
Comment (bigs) [inline source/getting_started.rst]: There -> These

File: source/getting_started.rst [hunk 13]
```diff
@@ -0,0 +1,528 @@
...
+   documentation
+
+.. code:: bash
```
Comment (bigs) [inline source/getting_started.rst]: `.. code-block:: js`

File: source/getting_started.rst [hunk 14]
```diff
@@ -0,0 +1,528 @@
...
+
+
+Make note of the ip, you'll need it:
+------------------------------------
+
+.. code:: bash
+
+        root@8c81350741d5:/# ifconfig
+        eth0      Link encap:Ethernet  HWaddr 02:42:ac:11:00:ac
+                  inet addr:172.17.0.175  Bcast:0.0.0.0  Mask:255.255.0.0
```
Comment (bigs) [inline source/getting_started.rst]: haha just occurring to me we should fuzz this IP to be something random/impossible

File: source/getting_started.rst [hunk 15]
```diff
@@ -0,0 +1,528 @@
...
+Visit the tracing dashboard
+----------------------------
+
+.. note:: Take note of your container's ip
+
+Concord
+
+.. code:: bash
+
+   http://172.17.0.175:9000/#/
```
Comment (bigs) [inline source/getting_started.rst]: fuzz ip here and below

File: source/getting_started.rst [hunk 16]
```diff
@@ -0,0 +1,528 @@
...
+
+.. code:: bash
+
+   root@68241956c4b6:/rb-getting-started#
+          concord deploy --config-file \
+          word_counter_deploy.json --zookeeper=127.0.0.1:2181
+
+
+
+You are done! :)
```
Comment (bigs) [inline source/getting_started.rst]: no smiley!

File: source/getting_started.rst (hunk 1 above)
Comment (senior7515) [inline source/getting_started.rst]: Good idea

File: source/getting_started.rst (hunk 2 above)
Comment (senior7515) [inline source/getting_started.rst]: where yo ?

File: source/getting_started.rst (hunk 3 above)
Comment (senior7515) [inline source/getting_started.rst]: good diea

File: source/getting_started.rst (hunk 5 above)
Comment (senior7515) [inline source/getting_started.rst]: not sure what you mean by this.

Comment (senior7515) [inline source/getting_started.rst]: go it.

File: source/getting_started.rst (hunk 9 above)
Comment (senior7515) [inline source/getting_started.rst]: i wanted the link explicitly. happy to make it a link too.

File: source/getting_started.rst (hunk 12 above)
Comment (senior7515) [inline source/getting_started.rst]: done.

File: source/getting_started.rst (hunk 15 above)
Comment (senior7515) [inline source/getting_started.rst]: why.. not sure this adds any value.

File: source/getting_started.rst (hunk 16 above)
Comment (senior7515) [inline source/getting_started.rst]: emoji?

File: source/getting_started.rst (hunk 1 above)
Comment (senior7515) [inline source/getting_started.rst]: This is solved  later.

**Pull request merged**